ionoscloud/resource_networkloadbalancer.go

```diff
@@ -45,6 +46,7 @@ func resourceNetworkLoadBalancer() *schema.Resource {
 					"listenerLan must be a customer reserved IP for the public load balancer and private IP " +
 					"for the private load balancer.",
 				Optional: true,
+				// Computed: true,
```

Comment:
do we need to keep this?

Reply:
it's an unrelated typo, i've reverted it